src/applications/personalization/dashboard-2/components/IconCTALink.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const IconCTALink = ({
+  ariaLabel,
+  href,
+  text,
+  icon,
```

Comment:
I think it'd be super helpful to mention that `icon` should be a valid [Font Awesome 5 icon name](https://fontawesome.com/icons?d=gallery&p=1&m=free)

Reply:
Added a comment!